BasicGates/BasicGates.ipynb

```diff
@@ -546,7 +546,52 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 2.4. Toffoli gate\n",
+    "### Task 2.4. Two-qubit gate - 4\n",
+    "\n",
+    "**Input:** Two unentangled qubits (stored in an array of length 2) in an arbitrary two-qubit state ${\\color{blue}\\alpha} |00\\rangle + {\\color{blue}\\beta} |01\\rangle + \\gamma |10\\rangle + \\delta |11\\rangle$.\n",
+    "\n",
+    "\n",
+    "**Goal:**  Change the two-qubit state to ${\\color{red}\\beta} |00\\rangle + {\\color{red}\\alpha} |01\\rangle + \\gamma |10\\rangle + \\delta |11\\rangle$.\n",
+    "\n",
+    "> This task can be solved using one library function; as an exercise, try to express the solution using several (possibly controlled) Pauli gates.  \n",
+    ">\n",
+    "> If you can't come up with a solution, or if you have solved the task, read about the relevant library function: \n",
+    "  <details> \n",
+    "    <summary><b>Click here.</b></summary>\n",
+    "    `ControlledOnInt` function can generate a controlled gate, which is applied only when the control qubit is in the $|0\\rangle$ state. </br>\n",
+    "    For example, `ControlledOnInt(0, H)([qs[0]], qs[1])` applies the Hadamard gate to the `qs[1]` qubit, when the control qubit `qs[0]` is in the $|0\\rangle$ state.\n",
+    "    </br></br>\n",
```

Comment:
Minor suggestion. <br> is an empty tag in html. [Doc link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br)
```suggestion
    "    <br><br>\n",
```